lib/linguist/languages.yml

```diff
@@ -44,6 +44,16 @@ AGS Script:
   tm_scope: source.c++
   ace_mode: c_cpp
 
+Alex Haskell:
+  type: programming
+  group: Haskell
```

Comment:
Currently, Alex Haskell files are showing up as Logos in the majority of these repositories
![screen shot 2015-06-10 at 3 46 55 am](https://cloud.githubusercontent.com/assets/2998727/8077387/61047e80-0f23-11e5-8e8b-e267736f9421.png)